spark-2.0-on-suse/azuredeploy.json

```diff
@@ -23,6 +23,57 @@
 	  "minValue": 1,
       "maxValue": 50
     },
+    "sparkSlavesSize": {
```

Comment:
If there is a pivot in machine size for this particular sample that's interesting we can always add to the readme...